### PR TITLE
Resource image profile acc test

### PR DESCRIPTION
**Pull request by Prativa20**
Signed-off-by: Prativa Bawri <bawrip@bawrip-a02.vmware.com>

### CODE REVIEW

File: vra/resource_image_profile_test.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/vmware/vra-sdk-go/pkg/client/image_profile"
+)
+
+func TestAccVRAImageProfileBasic(t *testing.T) {
+	rInt := acctest.RandInt()
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheckMachine(t) },
```
Comment (dmettem) [inline vra/resource_image_profile_test.go]: I think **testAccPreCheckMachine()** requires flavor, and project besides image. The test uses only the cloud account name and image. Please create a separate zoom with only those that are required for creating an Image Profile.

File: vra/resource_image_profile_test.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+	name := os.Getenv("VRA_AWS_CLOUD_ACCOUNT_NAME")
+	image := os.Getenv("VRA_IMAGE")
+	return fmt.Sprintf(`
+	data "vra_cloud_account_aws" "my-cloud-account" {
+		name = "%s"
+	  }
+
+data "vra_region" "us-east-1-region" {
+    cloud_account_id = data.vra_cloud_account_aws.my-cloud-account.id
+    region = "us-east-1"
```
Comment (dmettem) [inline vra/resource_image_profile_test.go]: Hard-coding the region name enforces the cloud account provided had that region selected already. Can we parameterize this?

File: vra/resource_image_profile_test.go (hunk 1 above)
Comment (Prativa20) [reply vra/resource_image_profile_test.go]: done!

File: vra/resource_image_profile_test.go (hunk 2 above)
Comment (Prativa20) [reply vra/resource_image_profile_test.go]: added variable for region

**Pull request merged**